packages/DatePicker/src/components/ShortcutPanel/ShortcutPanel.js

```diff
@@ -119,22 +119,34 @@ function ShortcutPanel(props) {
         </div>
         <div css={yearListStyles}>
           <div css={columnHeaderStyles} isYear>
-            <Button.Icon onClick={handleClickPrev} kind="minor" size="small">
+            <Button.Icon isSemantic={false} onClick={handleClickPrev} kind="minor" size="small">
               <ArrowLeft color={tokens.textColor.icon} />
             </Button.Icon>
             {I18n.t("datePicker.year")}
-            <Button.Icon onClick={handleClickNext} kind="minor" size="small">
+            <Button.Icon isSemantic={false} onClick={handleClickNext} kind="minor" size="small">
               <ArrowRight color={tokens.textColor.icon} />
             </Button.Icon>
           </div>
           <div role="group">{renderYearList()}</div>
         </div>
       </div>
       <div css={actionBarStyles}>
-        <Button kind="primary" onClick={handleConfirm} size="small" data-pka-anchor="datepicker.calendar.apply">
+        <Button
+          isSemantic={false}
+          kind="primary"
+          onClick={handleConfirm}
+          size="small"
+          data-pka-anchor="datepicker.calendar.apply"
+        >
           {I18n.t("actions.apply")}
         </Button>
-        <Button kind="minor" onClick={onCancel} size="small" data-pka-anchor="datepicker.calendar.cancel">
+        <Button
+          isSemantic={false}
+          kind="minor"
+          onClick={onCancel}
+          size="small"
+          data-pka-anchor="datepicker.calendar.cancel"
+        >
```

Comment:
Just changed all buttons to use `isSemantic={false}` in this file

Reply:
I know we've discussed setting the `isSemantic={false}` in a global hook for our styled components  (so that it is a default) however until that is setup i'm setting it on the individual buttons